README.adoc

```diff
@@ -214,7 +219,7 @@ Set to `1` to allow the service to run without actually writing keys or OS image
 == Using rpi-sb-provisioner
 `rpi-sb-provisioner` is composed of three `systemd` services that are triggered by the connection of a device in RPIBoot mode to a USB port. With `rpi-sb-provisioner` configured to your requirements, all that is therefore required is to connect your target Raspberry Pi device in RPIBoot mode.
```

Comment:
Interestingly, all of our [documentation](https://www.raspberrypi.com/documentation/) capitalises it as `rpiboot` or `RPIBOOT`, this is the first time I've seen it capitalised as `RPIBoot`.
```console
raspberrypi/documentation/develop$ git grep -i rpiboot | grep -v rpiboot | grep -v RPIBOOT
raspberrypi/documentation/develop$ 
```